docs/usage/types.md

```diff
@@ -719,6 +719,25 @@ _(This script is complete, it should run "as is")_
 
 Where `Field` refers to the [field function](schema.md#field-customisation).
 
+### Arguments to `constr`
+The following arguments are available when using the `constr` type function
+
+-  `strip_whitespace: bool = False`: removes leading and trailing whitespace
```

Comment:
```suggestion
- `strip_whitespace: bool = False`: removes leading and trailing whitespace
```
Same under.
Also maybe better to add type and default value for `max_length` and `curtail_length` to be homogeneous

Reply:
I meant to set only 1 space after `-` instead of 2 `strip_whitespace`.
Without space at all it won't be interpreted correctly

Reply:
Other than that looks good to me :) I guess we should to do the same for `confloat`, `conint`, `condecimal`, `conset` and `conlist` in another PR

Reply:
> Other than that looks good to me :) I guess we should to do the same for `confloat`, `conint`, `condecimal`, `conset` and `conlist` in another PR

I guess I'd do this in another PR once this is settled :)